### PR TITLE
Admin functionalities

### PR DESCRIPTION
**Pull request by sepehr9877**
adding admin templates and functionalities like brows job offers and application and update and delete them

### CODE REVIEW

**Review by nioushamahjoub — APPROVED** [review]

File: src/main/java/com/soen6011/SOEN6011/Controller/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		model.addAttribute("joboffers",joboffers);
+		return "admin_track_posts";
+	}
+	@GetMapping("/applications")
+	public String admin_applications(Model model) {
+		List<Applications> applications=this.applicationRepository.findAll();
+		model.addAttribute("Applications",applications);
+		return "admin_track_application";
+	}
+	@GetMapping("/deleteapplication/{id}")
```
Comment (MajorTreble) [inline src/main/java/com/soen6011/SOEN6011/Controller/AdminController.java:69]: Did you consider post?

File: src/main/java/com/soen6011/SOEN6011/Service/AdminService.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+public class AdminService {
+
+	private final JobOfferRepository jobOfferRepository;
+	
+	public AdminService(JobOfferRepository jobOfferRepository) {
+		this.jobOfferRepository=jobOfferRepository;
+	}
+	public Joboffers getJobOfferById(int id) {
+		
+		return this.jobOfferRepository.findById(id).get();
```
Comment (MajorTreble) [inline src/main/java/com/soen6011/SOEN6011/Service/AdminService.java:18]: This seems like alot for one line of code

File: src/main/java/com/soen6011/SOEN6011/model/Joboffers.java [hunk 3]
```diff
@@ -4,15 +4,18 @@
 
 import com.soen6011.SOEN6011.Enums.JobOfferStatus;
 
+
 import jakarta.persistence.*;
 
+
 @Entity
 public class Joboffers {
```
Comment (MajorTreble) [inline src/main/java/com/soen6011/SOEN6011/model/Joboffers.java:12]: Capital on Offers

File: src/main/resources/static/Design/styles/add-job-page.css
Comment (MajorTreble) [inline src/main/resources/static/Design/styles/add-job-page.css:1]: Does each page need a css?

**Comment by MajorTreble** [comment]
No unit tests
No javadoc

**Pull request merged**